client/ayon_core/plugins/publish/collect_anatomy_instance_data.py

```diff
@@ -407,8 +407,10 @@ def _fill_folder_data(self, instance, project_entity, anatomy_data):
             anatomy_data["hierarchy"] = hierarchy
 
             parent_name = project_entity["name"]
+            parents = []
```

Comment:
It seems valid to always have parents filled with at least project, right? Since we are converting parent_name from project name in case there is no parent at all it would be fair to also add it. In this case `{folder[parent[-1]]}`n would always work.

Reply:
I really don't think project should be there. But the use-case is that they would like to be able to use `{folder[parents][0]}` and `{folder[parents][1]}` not really `-1`.

Reply:
I don't think project name should be there too - the reason why is that you should be explicit enough if you want to have project name there (and you can access it with another key).

So if you have asset folder directly under project, `{folder[parents][0]}` will not work if I am not mistaken.

Reply:
> So if you have asset folder directly under project, {folder[parents][0]} will not work if I am not mistaken.

Yes, it won't.